lib/util/files/finder.js

```diff
@@ -1,59 +1,40 @@
-let { readdir, stat } = require("fs/promises");
-let path = require("path");
+let { readdir, stat } = require("node:fs/promises");
+let path = require("node:path");
 
 exports.FileFinder = class FileFinder {
-	constructor(directory, { skipDotfiles, filter = () => true } = {}) {
+	constructor(directory, { skipDotfiles, filter } = {}) {
 		this.directory = directory;
```

Comment:
```suggestion
	constructor(root, { skipDotfiles, filter } = {}) {
		this._root = root;
```

It doesn't look like the `directory` property is used by anyone; I've checked faucet-static and faucet-images.